resources/options.html

```diff
@@ -9,6 +9,7 @@
     <script src="jquery.min.js"></script>
     <script src="materialize.min.js"></script>
     <script src="options.js"></script>
+    <!--TODO: add the do not watch options -->
```

Comment:
Oh wait, I'm sorry, I had to continue reading. I understand what you have done.
My bad.